editor/scene_tree_dock.cpp

```diff
@@ -2663,12 +2663,17 @@ void SceneTreeDock::_new_scene_from(String p_file) {
 	Node *base = selection.front()->get();
 
 	HashMap<const Node *, Node *> duplimap;
+	HashMap<const Node *, Node *> inverse_duplimap;
 	Node *copy = base->duplicate_from_editor(duplimap);
 
+	for (KeyValue item : duplimap) {
+		inverse_duplimap[item.value] = (Node *)item.key;
```

Comment:
```suggestion
		inverse_duplimap[item.value] = item.key;
```
This should be unnecessary.